WhoIsThatServer.Storage/UnitTests/TargetElementControllerTests.cs

```diff
@@ -0,0 +1,100 @@
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FakeItEasy;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using Shouldly;
+using WhoIsThatServer.Storage.Controllers;
+using WhoIsThatServer.Storage.Helpers;
+using WhoIsThatServer.Storage.Models;
+
+namespace WhoIsThatServer.Storage.UnitTests
+{
+    public class TargetElementControllerTests
+    {
+        [Test]
+        public async Task Post_ShouldCallHelper()
+        {
+            //Arrange
+            var expectedId = 1;
+            var expectedHunterPersonId = 10;
+            var expectedPreyPersonId = 11;
+            var expectedIsHunted = false;
+
+            var expectedTargetElement = new TargetElement()
+            {
+                Id = expectedId,
+                HunterPersonId = expectedHunterPersonId,
+                PreyPersonId = expectedPreyPersonId
+            };
+            
+            var fakeTargetElementHelper = A.Fake<ITargetElementHelper>();
+            A.CallTo(() =>
+                fakeTargetElementHelper.InsertNewTargetElement(expectedId, expectedHunterPersonId,
+                    expectedPreyPersonId, expectedIsHunted)).Returns(expectedTargetElement);
+
+            var targetElementController = new TargetElementController()
+            {
+                TargetElementHelper = fakeTargetElementHelper,
+                Request = new HttpRequestMessage()
```

Comment:
Why do you need request in the constructor?

Reply:
Can't remember, I did that when I wrote tests for another controller and then for some reason I added it, now I'm not sure as I am not using it

Reply:
If I leave it blank, it fails and says that "Request must not be empty" or something like that